ui/src/App.tsx

```diff
@@ -34,26 +42,24 @@ interface DisplayedEventData {
   updatedAt: string;
 }
 
+interface formEntryProps {
```

Comment:
I think it's convention to capitalize interface names, so this could be `FormEntryProps`